packages/desktop-client/src/components/ManageRules.js

```diff
@@ -588,20 +588,34 @@ export default function ManageRules({
   }, []);
 
   function onCreateRule() {
+    let rule = {
+      id: null,
+      stage: null,
+      conditions: [
+        {
+          field: 'payee',
+          op: 'is',
+          value: null,
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏could you please move the expression here? It will improve the readability.
```suggestion
          value: payeeId || null,
```